src/clj/game/cards/assets.clj

```diff
@@ -681,7 +697,9 @@
     :events {:corp-turn-begins ability}})
 
    "Melange Mining Corp."
-   {:abilities [{:cost [:click 3] :effect (effect (gain :credit 7)) :msg "gain 7 [Credits]"}]}
+   {:abilities [{:cost [:click 3] :effect (effect (do (gain :credit 7)
+                                                      (trigger-event state side :corp-spend-click (:cid card) 3)))
```

Comment:
Are you aware that `effect` injects `state side` as the first two arguments? So this would look like:

```
(effect (gain :credit 7)
        (trigger-event :corp-spend-click (:cid card) 3))
```

But I agree with @nealterrell that this seems unnecessary. Can we not check to see if a cost contains clicks when resolving an ability and trigger the event there, in general, like you do with the other events?